.gitignore

```diff
@@ -5,6 +5,7 @@ target
 .direnv
 
 *.zip
+target_pol/docs*
```

Comment:
You can ignore the entire `target_pol` directory, not just the `target_pol/docs` subdirectory.